Lib/test/test_int.py

```diff
@@ -906,6 +906,18 @@ def test_pylong_misbehavior_error_path_from_str(
             with self.assertRaises(RuntimeError):
                 int(big_value)
 
+    def test_pylong_roundtrip(self):
+        from random import randrange
+        bits = 5000
+        while bits <= 1_000_000:
+            bits += randrange(-100, 101) # break bitlength patterns
+            hibit = 1 << (bits - 1)
+            n = hibit + randrange(hibit)
```

Comment:
'getrandbits(bits - 1)` can be slightly faster.
```suggestion
            n = hibit | getrandbits(bits - 1)
```

Reply:
The savings would be trivial compared to the two bigint base conversions that follow it, and the enclosing loop doesn't even go around 10 times ... but, ya, I think it's a little _clearer_, so I'll accept it. Thanks!